Schedule-API/Schedule-API/Controllers/EventController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using ScheduleAPI.Models;
+using ScheduleAPI.Repositories;
+using ScheduleAPI.ViewModels;
+
+namespace ScheduleAPI.Controllers
+{
+    public class EventController : Controller
```

Comment:
@elekamate  So this controller should have the CRUD methods implemented.
I'm expecting something like: 

- GET: api/Events

- GET: api/Events/id

- POST: api/Events

- PUT: api/Events/id

- DELETE: api/Events/id

All of them should be `IActionResult` and return at least the correct `StatusCode`